easybuild/easyblocks/i/imkl_fftw.py

```diff
@@ -61,11 +61,6 @@ def make_module_step(self, *args, **kwargs):
         """
         return super(EB_imkl, self).make_module_step(*args, **kwargs)
 
-    def make_module_extra(self):
-        """Custom extra variables to set in module file"""
-        # bypass extra module variables for imkl
-        return super(EB_imkl, self).make_module_extra()
-
```

Comment:
These lines should be kept, otherwise the module of imkl-fftw gets stuff that belongs to imkl like `MKLROOT`. Please revert.